### PR TITLE
Adding Naukri.com Related Website Set

### PR DESCRIPTION
**Pull request by naukri-engineering**
Adding Naukri.com RWS Set

### CODE REVIEW

File: related_website_sets.JSON [hunk 1]
```diff
@@ -836,6 +836,18 @@
         "https://gnttv.com": " Good News Today is a website that focuses on the bright side of life and showcases stories of triumphs and positivity.",
         "https://indiatodayne.in": "India Today NE is dedicated to covering news and views of India’s north-eastern region which includes eight states—Assam, Arunachal Pradesh, Nagaland, Manipur, Mizoram, Tripura, Meghalaya and Sikkim"
     }
-  }
+  },
+ {
+        "contact": "naukri.comapp@gmail.com",
+        "primary": "https://naukri.com",
```
Comment (hamadilubinga) [inline related_website_sets.JSON:842]: Améthyst pierre magique c'est pour la sagesse et l'intelligence.

**Review by hamadilubinga — COMMENTED** [review]
Améthyst pierre magique c'est pour la sagesse et l'intelligence.

**Review by guptadivyansh2392 — COMMENTED** [review]
Changes looks fine for Naukri Domain

**Review by shuranhuang — APPROVED** [review]
LGTM

**Review by hamadilubinga — COMMENTED** [review]
Bougie orange c'est pour la prospérité et de la chance.

**Comment by hamadilubinga** [comment]
Perles de Grand-mère c'est pour l'amour et de la chance.

File: related_website_sets.JSON [hunk 2]
```diff
@@ -836,6 +836,18 @@
         "https://gnttv.com": " Good News Today is a website that focuses on the bright side of life and showcases stories of triumphs and positivity.",
         "https://indiatodayne.in": "India Today NE is dedicated to covering news and views of India’s north-eastern region which includes eight states—Assam, Arunachal Pradesh, Nagaland, Manipur, Mizoram, Tripura, Meghalaya and Sikkim"
     }
-  }
```
Comment (Parasos) [inline related_website_sets.JSON:839]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
172012736291

File: related_website_sets.JSON [hunk 3]
```diff
@@ -836,6 +836,18 @@
...
+  },
+ {
+        "contact": "naukri.comapp@gmail.com",
+        "primary": "https://naukri.com",
+        "associatedSites": [
+           "https://ambitionbox.com",
+           "https://infoedgeindia.com"
+        ],
```
Comment (Parasos) [inline related_website_sets.JSON:846]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
Submission_Guidelines.mdhttps://accounts.google.com/v3/signin/rejected?continue=https://chrome.google.com/sync&ddm=0&dsh=S1191280729:1720127362912122&flowEntry=ServiceLogin&flowName=GlifWebSignIn&followup=https://chrome.google.com/sync&ifkv=AS5LTAQLy8WVMkj4746iRWpK2EpU2J76SrSyxREUtebrkY6wLHrHcu9QOy6HAyl1EO8WqITixPMZ5g

**Comment by Parasos** [comment]
good

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

File: related_website_sets.JSON [hunk 4]
```diff
@@ -836,6 +836,18 @@
         "https://gnttv.com": " Good News Today is a website that focuses on the bright side of life and showcases stories of triumphs and positivity.",
         "https://indiatodayne.in": "India Today NE is dedicated to covering news and views of India’s north-eastern region which includes eight states—Assam, Arunachal Pradesh, Nagaland, Manipur, Mizoram, Tripura, Meghalaya and Sikkim"
     }
-  }
+  },
+ {
+        "contact": "naukri.comapp@gmail.com",
+        "primary": "https://naukri.com",
+        "associatedSites": [
```
Comment (Parasos) [inline related_website_sets.JSON:843]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
https://support.google.com/accounts/answer/7675428?hl=ru

File: related_website_sets.JSON [hunk 5]
```diff
@@ -836,6 +836,18 @@
...
+           "https://ambitionbox.com",
+           "https://infoedgeindia.com"
+        ],
+        "rationaleBySite": {
+           "https://ambitionbox.com": "Ambition box associated url",
+           "https://infoedgeindia.com": "InfoEdge India website"
```
Comment (Parasos) [inline related_website_sets.JSON:849]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

File: related_website_sets.JSON [hunk 6]
```diff
@@ -836,6 +836,18 @@
...
+  },
+ {
+        "contact": "naukri.comapp@gmail.com",
+        "primary": "https://naukri.com",
+        "associatedSites": [
+           "https://ambitionbox.com",
+           "https://infoedgeindia.com"
+        ],
+        "rationaleBySite": {
+           "https://ambitionbox.com": "Ambition box associated url",
```
Comment (Parasos) [inline related_website_sets.JSON:848]: related_website_sets.JSONSubmission_Guidelines.mdhttps://accounts.google.com/v3/signin/rejected?continue=https://chrome.google.com/sync&ddm=0&dsh=S1191280729:1720127362912122&flowEntry=ServiceLogin&flowName=GlifWebSignIn&followup=https://chrome.google.com/sync&ifkv=AS5LTAQLy8WVMkj4746iRWpK2EpU2J76SrSyxREUtebrkY6wLHrHcu9QOy6HAyl1EO8WqITixPMZ5g

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

File: related_website_sets.JSON (hunk 5 above)
Comment (Parasos) [inline related_website_sets.JSON:849]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
- related_website_sets.JSON

**Comment by Parasos** [comment]
good

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSONhttps://support.google.com/accounts/answer/7675428?hl=ruhttps://accounts.google.com/v3/signin/rejected?continue=https://chrome.google.com/sync&ddm=0&dsh=S1191280729:1720127362912122&flowEntry=ServiceLogin&flowName=GlifWebSignIn&followup=https://chrome.google.com/sync&ifkv=AS5LTAQLy8WVMkj4746iRWpK2EpU2J76SrSyxREUtebrkY6wLHrHcu9QOy6HAyl1EO8WqITixPMZ5g&rhlk=js&rrk=47

**Comment by Parasos** [comment]
951973947d06cd936df401629f9cd13b42cd9011

File: related_website_sets.JSON [hunk 7]
```diff
@@ -836,6 +836,18 @@
...
+           "https://infoedgeindia.com": "InfoEdge India website"
+        }
```
Comment (Parasos) [inline related_website_sets.JSON:850]: related_website_sets.JSON951973947d06cd936df401629f9cd13b42cd9011

**Review by Parasos — COMMENTED** [review]
Submission_Guidelines.mdhttps://accounts.google.com/v3/signin/rejected?continue=https://chrome.google.com/sync&ddm=0&dsh=S1191280729:1720127362912122&flowEntry=ServiceLogin&flowName=GlifWebSignIn&followup=https://chrome.google.com/sync&ifkv=AS5LTAQLy8WVMkj4746iRWpK2EpU2J76SrSyxREUtebrkY6wLHrHcu9QOy6HAyl1EO8WqITixPMZ5g

**Review by Parasos — COMMENTED** [review]
"contact": "naukri.comapp@gmail.com",naukri.comapp@gmail.com

**Pull request merged**